src/ast/passes/codegen_llvm.cpp

```diff
@@ -3588,7 +3588,11 @@ void CodegenLLVM::createPrintNonMapCall(Call &call, int id)
 void CodegenLLVM::generate_ir()
 {
   assert(state_ == State::INIT);
-  generate_maps(bpftrace_.resources);
+
+  auto cra = CodegenResourceAnalyser(root_, bpftrace_.config_);
```

Comment:
super nit: not a big fan of these abbreviations - much prefer something like `codegen_resources`